tachyon_cli/README

```diff
@@ -0,0 +1,104 @@
+# Tachyon CLI
```

Comment:
Nit(s):

* Can this be renamed to `README.md`?
* Can we have a newline at the end of this file?
* Can we link to this in the main repo `README.md`?

Otherwise, awesome work on this!

Reply:
Fixed!